src/storage/src/source/postgres/snapshot.rs

```diff
@@ -334,7 +332,7 @@ pub(crate) fn render<G: Scope<Timestamp = MzOffset>>(
                 let desc = match verify_schema(oid, expected_desc, &upstream_info) {
                     Ok(()) => expected_desc,
                     Err(err) => {
-                        raw_handle.give(data_cap, ((oid, Err(err)), MzOffset::minimum(), 1)).await;
+                        raw_handle.give(&data_cap_set[0], ((oid, Err(err)), MzOffset::minimum(), 1)).await;
```

Comment:
i would prefer an extension trait impl on `CapabilitySet` where if `T: TotalOrder` you can get the `time` out, but that can be left as a followup